Firmware/Klipper/Voron_2.1/Klipper/Configurations/VORON2_stock_250_printer.cfg

```diff
@@ -205,51 +254,114 @@ max_power: 1.0
 #   will be scaled between zero and max_power (for example, if
 #   max_power is .9 and a fan speed of 80% is requested then the fan
 #   power will be set to 72%). The default is 1.0.
-#shutdown_speed: 0
+shutdown_speed: 0
 #   The desired fan speed (expressed as a value from 0.0 to 1.0) if
 #   the micro-controller software enters an error state. The default
 #   is 0.
-#cycle_time: 0.010
+cycle_time: 0.010
 #   The amount of time (in seconds) for each PWM power cycle to the
 #   fan. It is recommended this be 10 milliseconds or greater when
 #   using software based PWM. The default is 0.010 seconds.
-#hardware_pwm: True
+hardware_pwm: false
 #   Enable this to use hardware PWM instead of software PWM. The
 #   default is False.
 kick_start_time: 0.500
 #   Time (in seconds) to run the fan at full speed when first enabling
 #   it (helps get the fan spinning). The default is 0.100 seconds.
+#   See the "fan" section in example.cfg for a description of the
+#   above parameters.
 
 [heater_fan hotend_fan]
 # Hotend fan
+#   See the "heater_fan" section in example-extras.cfg for a complete
+#   description of this sections parameters.
 pin: z:ar9
 #   D9 on mcu_z
 max_power: 1.0
+shutdown_speed: 0.75
```

Comment:
Klipper does not accept this vlaue.

Recv: // shutdown value must be 0.0 or 1.0 on soft pwm
Recv: // Once the underlying issue is corrected, use the "RESTART"
Recv: // command to reload the config and restart the host software.
Recv: // Printer is halted
Recv: !! shutdown value must be 0.0 or 1.0 on soft pwm

Reply:
Noted.  Thank you.  I'll update, accordingly